src/library/simplified/extension/currency-code.js

```diff
@@ -1,96 +1,6 @@
-        b["currencyCode"] = b["entity.multiItemSearch.multiItemSearchResults.offers.0.packagePrice.packageTotalPrice.currency"];
-    }
-}
-else if(utag.isCarSR() && b["entity.carSearch.lowestOfferDailyPrice.currency"] != undefined){
-    b["currencyCode"] = b["entity.carSearch.lowestOfferDailyPrice.currency"];
-}
-else if(utag.isCarUDP() && b["entity.carDetails.totalPrice.currency"] !== undefined) {
-	b["currencyCode"] = b["entity.carDetails.totalPrice.currency"];
-}
 
-//LX
-else if (utag.isLXS() && b["entity.activities.activitySearchResults.cheapestPrice.currency"] != undefined){
-    b["currencyCode"] = b["entity.activities.activitySearchResults.cheapestPrice.currency"];
-}
-
-//Rails Path
-else if (utag.isRailSearchResults() && b['entity.railSearch.searchResults.railLegs.0.railOfferItems.0.price.currency'] != undefined) {
-    b["currencyCode"]  = b['entity.railSearch.searchResults.railLegs.0.railOfferItems.0.price.currency'];
-}
-
-else if (utag.isRailRateDetails() && b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'] != undefined) {
-    b['currencyCode'] = b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'];
-}
-
-//Common data element
-else if((utag.isCarPymt() || utag.isCarCO() || utag.isHCO() || utag.isHPymt()
-    || utag.isCruisePymt() || utag.isCruiseCO() || utag.isLXCO() || utag.isPCO() || utag.isPCO_Mobile() || utag.isMCO()) && typeof b["entity.checkout.cartTotal.currency"] != "undefined"){
-    b["currencyCode"] = b["entity.checkout.cartTotal.currency"];
-}
-else if(utag.isLXCO() && b.entity.checkout.cartTotal.currency){
-    b["currencyCode"] = b.entity.checkout.cartTotal.currency;
-}
-else if(utag.isCarCO() && typeof b["entity.tripDetails.totalPrice"] != "undefined"){
-    b['currencyCode'] = b["entity.tripDetails.totalPrice.currency"] ? b["entity.tripDetails.totalPrice.currency"] : "";
-}
-else if(utag.isItinPage() && typeof b["entity.tripDetails.totalPrice"] != "undefined"){
-    b['currencyCode'] = b["entity.tripDetails.totalPrice.currency"] ? b["entity.tripDetails.totalPrice.currency"] : "";
-}
-else if(utag.is3pp()){
-    b['currencyCode'] = b["entity.checkout.currency"];
-}
-else if(utag.isPCF() && typeof b['entity.packageSearch.results.offers.0.packagePrice.packageTotalPrice.currency'] !== undefined) {
-    b['currencyCode'] = b['entity.packageSearch.results.offers.0.packagePrice.packageTotalPrice.currency'];
-}
-else if(utag.isLXGT_CO() && b.entity.checkout.cartTotal.currency){
-    b['currencyCode'] = b.entity.checkout.cartTotal.currency;
-}
-
-//All Slim confirmation pages
-if(b.pageInfo.pageName && b.pageInfo.pageName.indexOf("Checkout.Confirmation.Slim") > -1 && b.entity.checkout.currency){
+//All confirmation pages including slim.
+if(b.pageInfo.pageName && b.pageInfo.pageName.indexOf("Checkout.Confirmation") > -1 && b.entity.checkout.currency){
```

Comment:
Will this new line set currency for all other pages such as HSR, HIS ?

Reply:
Only pages that contains "Checkout.Confirmation" such as:
page.Checkout.Confirmation
page.Flight.Checkout.Confirmation
page.Hotel.Checkout.Confirmation
page.Car.Checkout.Confirmation
page.LX.Checkout.Confirmation
etc...